hathor/transaction/aux_pow.py

```diff
@@ -19,7 +19,8 @@
 logger = get_logger()
 
 
-MAX_MERKLE_PATH_LENGTH: int = 12
+OLD_MAX_MERKLE_PATH_LENGTH: int = 12
+NEW_MAX_MERKLE_PATH_LENGTH: int = 15
```

Comment:
I think these parameters should be moved to settings.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/910/commits/f4b6182ad58640dc0a0f51cdeb967e178c298873